lz4/scripts/asplos21/chart_breakdown.py

```diff
@@ -0,0 +1,180 @@
+import argparse
```

Comment:
LZ4 has nothing to do with asplos - don't include these files
we can move the scripts around in a separate commit